google/cloud/storage/internal/policy_document_request.cc

```diff
@@ -342,7 +342,12 @@ std::map<std::string, std::string> PolicyDocumentV4Request::RequiredFormFields()
 }
 
 std::ostream& operator<<(std::ostream& os, PolicyDocumentV4Request const& r) {
-  return os << "PolicyDocumentRequest={" << r.StringToSign() << "}";
+  using nlohmann::json;
+  json j;
+  j["endpoint_authority"] = r.endpoint_authority();
+
+  return os << "PolicyDocumentRequest={" << std::move(j.dump()) << ","
```

Comment:
Consider:

```suggestion
  return os << "PolicyDocumentRequest={endpoint_authority=" << r.endpoint_authority() << ","
```

For future reference, you can initialize `nlohmann::json` objects using:

```cc
  auto const j = nlohmann::json{{"endpoint_authority", r.endpoint_authority()}, {"foo", true}, {"bar", 12345}};
```

And even create recursive objects.